designcompose/src/main/java/com/android/designcompose/proto/DimensionRect.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose.proto
+
+import com.android.designcompose.definition.element.dimensionRect
+import com.android.designcompose.serdegen.Dimension
+import com.android.designcompose.serdegen.DimensionRect
+import java.util.Optional
+
+fun Optional<DimensionRect>.getStart(): Dimension =
+    this.orElseThrow { NoSuchFieldException("Malformed data: DimensionRect unset") }.start.getDim()
```

Comment:
You might be able to do this as a property getter, so that you can just call `.start` instead of `.getStart()`.
```
var A.someProperty: Int
  get() = /* return something */
```